components/brave_wallet_ui/components/shared/create-placeholder-icon/index.tsx

```diff
@@ -132,19 +115,12 @@ export function withPlaceholderIcon<
       }
     }, [needsPlaceholder, asset?.contractAddress, asset?.name])
 
-    const remoteImage = React.useMemo(() => {
-      if (isRemoteURL) {
-        return isStorybook ? imgSrc || '' : `chrome://image?${imgSrc}`
-      }
-      return ''
-    }, [isRemoteURL, imgSrc])
-
     // render
     if (!asset) {
       return null
     }
 
-    const icon = nativeAssetLogo || (isRemoteURL ? remoteImage : asset?.logo)
+    const icon = nativeAssetLogo || asset?.logo
```

Comment:
Shouldn't we still prepend the URL in `chrome://image?` to the image URL for non NFT urls?

Reply:
yes, corrected it